calculator.py

```diff
@@ -0,0 +1,35 @@
+import datetime
+ 
+all_data = {
+    "Transaction Type": "",
+    "Origin or Destination": "",
+    "Amount": "",
+    "Date": ""
+}
+ 
+def data():
+    # input either income or expense
+    choose = input("Income or Expense? ")
+    all_data.update({"Transaction Type": choose})
+ 
+    # from/to, amount, date (optional)
+    choose2 = input("From or where to is this money going? ")
+    all_data.update({"Origin or Destination": choose2})
+    if choose == "Income" or choose == "Expense":
+        transactionAmount = input("How much money was involved in this transaction? ")
+        all_data.update({"Amount": transactionAmount})
```

Comment:
should cast to float type before saving otherwise you won't be able to compare it as an amount later